lib/cfg-lexer-subst.c

```diff
@@ -92,6 +92,8 @@ _track_string_state(CfgLexerSubst *self, CfgLexerStringTrackState last_state, co
       if (*p == '\'')
         return CLS_NOT_STRING;
       return CLS_WITHIN_QSTRING;
+    default:
```

Comment:
In this case we are in a state machine. It is a fatal error to enter to the default state. Please add g_assert_not_reached() into the default branch.